guide/src/building_and_distribution.md

```diff
@@ -225,6 +225,10 @@ The known complications are:
 
 If you encounter these or other complications when linking the interpreter statically, discuss them on [issue 416 on PyO3's Github](https://github.com/PyO3/pyo3/issues/416). It is hoped that eventually that discussion will contain enough information and solutions that PyO3 can offer first-class support for static embedding.
 
+### Import your module when embedding the CPython interpreter
```

Comment:
In general PyPy doesn't support embedding, so I don't think you need to make the distinction here.

```suggestion
### Import your module when embedding the Python interpreter
```